web_archiving/readme.md

```diff
@@ -49,7 +49,7 @@ Capabilities are evaluated as "out of the box" functionality. In cases where a s
 
 * _**WUI**:_ Whether the software offers a web-based user interface.
 
-* _**Library API**:_ Does it provide an explicit libary interface? (I.e., does it provide an API so that you can use it in your own program?)
+* _**Library API**:_ Does it provide an explicit libary interface? (i.e., does it provide an API so that you can use it in your own program?)
```

Comment:
@mhucka --

I'm used to writing in APA and Chicago style where i.e., and e.g., are: 
- used in parenthesis
- undercase
- followed by a comma

I couldn't find an example in either where i.e. or e.g. was led by a capital ([chicago](http://www.chicagomanualofstyle.org/16/ch13/ch13_sec007.html), [apa](http://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html)), and in both I believe lower case to be preferred. I'm happy to defer to another style if that is everyone's preference